client/my-sites/stats/site.jsx

```diff
@@ -260,6 +266,43 @@ class StatsSite extends Component {
 						</div>
 					</div>
 				</div>
+				<div className="stats__promo">
+					<div className="stats__promo-text">
+						{ isFreePlan && (
+							<>
+								<h2 className="stats__promo-title">{ translate( 'Start earning money now.' ) }</h2>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Start earning money now' )` *ES Score: 13*